src/main/java/interface_adapters/OutputBoundary.java

```diff
@@ -1,12 +1,12 @@
 package interface_adapters;
 
 public interface OutputBoundary {
-    public void update_Text(String line1, String line2, String line3, String line4);
-    public void update_Hp(int hp);
-    public void update_EssenceCnt(int cnt);
-    public void update_Artifact(int cnt);
-    public void update_Playerlocation(int[] location);
-    public void update_Win();
-    public void update_Dead();
-
+    void updateText(String line1, String line2, String line3, String line4);
+    void updateHp(int hp);
+    void updateEssenceCnt(int cnt);
+    void updateArtifact(int cnt);
+    void updatePlayerlocation(int[] location);
+    void updateWin();
+    void updateDead();
+    void updateMap(int[][] map);
```

Comment:
Now the name is perfectly Camel Case. If we merge it into the main, please check the people who use the interface and correct the names.